ui/imports/shared/popups/addaccount/panels/AccountAddressSelection.qml

```diff
@@ -185,20 +195,10 @@ StatusMenu {
                         icon.width: 16
                         icon.height: 16
                         onClicked: {
-                            Qt.openUrlExternally("https://etherscan.io/address/%1".arg(model.addressDetails.address))
```

Comment:
Clicking the external link button didn't work at all because of the MouseArea below covering this small button

Reply:
Not sure when/why did got broken, worked in the past.

Reply:
I don't really think this could ever work :laughing: 

![image](https://github.com/user-attachments/assets/1eeec12e-c576-4b9e-970f-9fd361af53b4)